api/diagrams/error-detail-nack.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd"><svg xmlns="http://www.w3.org/2000/svg" width="1092" height="678" xmlns:xlink="http://www.w3.org/1999/xlink"><source><![CDATA[participant Envoy as C
+participant Management Server as S
+
+C->S: {resource_names={A}}
+S->C: {resources={A}, version=1, nonce=1}
+C->S: {resource_names={A}, version=1, nonce=1 (this is an ACK)}
+C->S: {resource_names={A, B}, version=1, nonce=1}
+Note over C: adding subscription\nto B
```

Comment:
I think in all the "Note over"s that they are below the line they are referring to. E.g. here, the addition of subscription to B by the client happens in the line above when `{A, B}` is sent.

Reply:
Done, should be clearer now.